appengine/pubsub/app.yaml

```diff
@@ -16,10 +16,10 @@ runtime: ruby
 env: flex
 entrypoint: bundle exec ruby app.rb
 
-# [START gae_flex_pubsub_yaml]
+# [START gae_pubsub_yaml]
```

Comment:
Do you want to change this region_tag? You'll have to update all the docs.

Reply:
Yes, having one tag for both flex and standard simplifies the doc tagging.